.github/workflows/build-msvc.yml

```diff
@@ -136,8 +136,8 @@ jobs:
         shell: bash -eo pipefail -o igncr {0}
         run: |
           eval $(tools/msvs-promote-path)
-          make -j tests
+          make -j$(nproc) tests
```

Comment:
These days, this shouldn't be doing anything anyway (the testsuite components and libraries are built as part of the main build since #12706) - perhaps just delete the `-j` instead?

Reply:
Right.